### ISSUE TITLE
Jeep Trackhawk

### ISSUE THREAD

**Issue by CasperBoiRL** [fkk-cz/noire_vehicles#1211]
It’s a big massive SUV, so it should have a big massive 1000 trunk space. But, it only has 750.

**Comment by CasperBoiRL** [comment]
And the tahoe has 1000, so this should too.

**Comment by No-Yes1** [comment]
Fixed the tahoe back to SUV vehicle class. Thanks for bringing it to my attention. https://github.com/fkk-cz/noire_vehicles/commit/438280fd1cc780e3909b3171a769f98c36cf2b3f

**Comment by CasperBoiRL** [comment]
Yes, it’s an SUV, but all SUVS don’t have the same trunk space. Tell me how the durango has the same as the trackhawk, and it has a whole ass 3rd row. The trackhawk has tons of space, and should be 1k

**Comment by CasperBoiRL** [comment]
i didn’t do it on purpose and i only did it to 1 car. and y’all better not change it becaus that’s the only car that non supporters can use on their trailers for 1k storabhe

**Comment by Meelokbruvv** [comment]
Bro he already did change it the Tahoe is reclassed so watch at reset it's gonna be 750 trunk

**Comment by okxeon** [comment]
Why shouldn’t it be? Just because its the only current non supporter suv with 1k storage does not mean it must stay that way. There are other vehicles that can be put on the trailer to get 1k storage.

**Comment by CasperBoiRL** [comment]
That part doesn’t really matter, i’m just saying that it should be at 1k, just because of the size of it
